pupil-api/src/config.ts

```diff
@@ -9,12 +9,11 @@ const getEnvironment = () => {
 
 export = {
   PORT: process.env.PORT || '3003',
-  // autoMark true | false - Automatically mark the check data when we receive it: boolean
-  autoMark: process.env.hasOwnProperty('AUTO_MARK') ? toBool(process.env.AUTO_MARK) : true,
   Logging: {
     Express: {
-      UseWinston: process.env.EXPRESS_LOGGING_WINSTON || false
+      UseWinston: process.env.hasOwnProperty('EXPRESS_LOGGING_WINSTON') ? toBool(process.env.EXPRESS_LOGGING_WINSTON) : false
```

Comment:
I think simplifying this ternary to: `process.env.hasOwnProperty('EXPRESS_LOGGING_WINSTON') && toBool(process.env.EXPRESS_LOGGING_WINSTON)`

Reply:
Yes, that's okay if the default is to be false, but how would you default to true in the future?  I prefer leaving it as is, as it allows the default setting to be changed more easily.

Reply:
If the default value is to be changed in the future then yes leave it as it is.